man/ctags-client-tools.7.rst.in

```diff
@@ -462,6 +462,84 @@ client tool could distinguish them by the ``TAG_OUTPUT_EXCMD`` pseudo tag, it's
 probably there's no need to treat them differently, since "search for the
 nearest occurence from the line" gives good result on both schemes.
 
+JSON OUTPUT
+-----------
+Universal Ctags supports `JSON <https://www.json.org/>`_ (strictly
+speaking `JSON Lines <https://jsonlines.org/>`_) output format if the
+ctags executable is built with ``libjansson``.  JSON output goes to
+standard output by default.
+
+Format
+~~~~~~
+Each JSON line represents a tag.
+
+.. code-block:: console
+
+	$ ctags --extras=+p --output-format=json --fields=-s input.py
+	{"_type": "ptag", "name": "JSON_OUTPUT_VERSION", "path": "0.0", "pattern": "in development"}
+	{"_type": "ptag", "name": "TAG_FILE_SORTED", "path": "1", "pattern": "0=unsorted, 1=sorted, 2=foldcase"}
+	...
+	{"_type": "tag", "name": "Klass", "path": "/tmp/input.py", "pattern": "/^class Klass:$/", "language": "Python", "kind": "class"}
+	{"_type": "tag", "name": "method", "path": "/tmp/input.py", "pattern": "/^    def method(self):$/", "language": "Python", "kind": "member", "scope": "Klass", "scopeKind": "class"}
+	...
+
+A key not starting with ``_`` is mapped to a field of ctags.
+"``--output-format=json --list-fields``" options list the fields.
+
+A key starting with ``_`` represents meta information of the JSON
+line.  Currently only ``_type`` key is used. If the value for the key
+is ``tag``, the JSON line represents a normal tag. If the value is
+``ptag``, the line represents a pseudo-tag.
+
+The output format can be changed in the
+future. ``JSON_OUTPUT_VERSION`` pseudo-tag provides a change
+client-tools to handle the changes.  Current version is "0.0". A
+client-tool can extract the version with ``path`` key from the
+pseudo-tag.
+
+The JSON output format is newly designed and has no limitation found
+in the default tags file format.
+
+* The values for ``kind`` key are represented in long-name flags.
+  No one-letter is here.
+
+* scope kinds and scope names have distinguished keys: ``scope`` and ``scopeKind``.
```

Comment:
"Scope names and scope kinds ..."  (capitalize and change the order as ``scope`` and ``scopeKind``).

Reply:
I see. Thank you.